includes/process-donation.php

```diff
@@ -815,8 +815,10 @@ function give_register_and_login_new_user( $user_data = array() ) {
 	 */
 	do_action( 'give_insert_user', $user_id, $user_data );
 
-	// Login new user.
-	give_log_user_in( $user_id, $user_data['user_login'], $user_data['user_pass'] );
+	if ( true === (bool) apply_filters( 'give_log_user_in_on_register', true ) ) {
```

Comment:
@raftaar1191 Add documentation to this filter

Reply:
Done